bijlage03_Versienummering.md

```diff
@@ -60,7 +63,7 @@ Voor standaarden is relevant of een realisatie volgens de oude versie van een st
 Globale regels voor het bepalen van de impact op de versionering:
 
 - Toevoegen = Major
-- Wijzingen :
+- Wijzigingen :
```

Comment:
ik zou dit wijzigen in:
Wijzigingen = Major of Minor (afhankelijk van de verwachte impact van de wijziging)
```suggestion
- Wijzigingen = Major of Minor (afhankelijk van de verwachte impact van de wijziging)
```